data/remnant outfits.txt

```diff
@@ -359,6 +359,26 @@ outfit "Salvage Scanner"
 
 # Engines:
 
+outfit "Anvil-Class Engine"
+	category "Engines"
+	licenses
+		Remnant
+	"cost" 290000
+	thumbnail "outfit/tiny remnant engine"
+	"mass" 26
+	"outfit space" -26
+	"engine capacity" -26
+	"thrust" 13.5
+	"thrusting energy" 2.05
+	"thrusting heat" 1.95
+	"turn" 336.0
+	"turning energy" 1.05
+	"turning heat" 1.1
+	"flare sprite" "effect/remnant flare/small"
+		"frame rate" 5
+	"flare sound" "plasma tiny"
+	description "This tiny set of engines were designed specifically for craft so small that they to combine both thrusters and steering into one unit."
```

Comment:
`that they to combine` Seems like you messed up here.
How about `"This tiny set of engines were designed specifically for craft so small that they're unable to fit a normal set of engines, leading to both thrusters and steering being combined into one unit."`

Reply:
Fixed by Pointedstick's change.